_posts/en/newsletters/2023-11-29-newsletter.md

```diff
@@ -0,0 +1,132 @@
+  testing for their configuration.
+
+- [What is the relation between nLockTime and nSequence?]({{bse}}120256)
+  Antoine Poinsot and Pieter Wuille answer a series of Stack Exchange questions about
+  `nLockTime` and `nSequence` including the [relationship between the
+  two]({{bse}}120273), the [original purpose of `nLockTime`]({{bse}}120276), the [potential values of `nSequence`]({{bse}}120254) and
+  relationship to [BIP68]({{bse}}120320) and [`OP_CHECKLOCKTIMEVERIFY`]({{bse}}120259).
+
+- [What would happen if we provide to OP_CHECKMULTISIG more than threshold number (m) of signatures?]({{bse}}120604)
+  Pieter Wuille explains that while this was possible previously, the [BIP147][]
+  dummy stack element malleability soft fork no longer allows the additional
+  stack element in OP_CHECKMULTISIG and OP_CHECKMULTISIGVERIFY to be an arbitrary value.
+
+- [What is "(mempool) policy"?]({{bse}}120269)
+  Antoine Poinsot defines the terms policy and standardness in the context of
+  Bitcoin Core and provides a few examples. He also links to Bitcoin Optech's
+  [Waiting for confirmation][policy series] series on policy.
+
+- [What does Pay to Contract (P2C) mean?]({{bse}}120362)
+  Vojtěch Strnad describes [Pay-to-Contract (P2C)][topic p2c] and links to the
+  [original proposal][p2c paper].
+
+- [Can a non-segwit transaction be serialized in the segwit format?]({{bse}}120317)
+  Pieter Wuille points out that while several older versions of Bitcoin Core
+  unintentionally allowed the extended serialization format for non-segwit
+  transactions, [BIP144][] specifies non-segwit transactions must use the old
+  serialization format.
+
+## Releases and release candidates
+
+*New releases and release candidates for popular Bitcoin infrastructure
+projects.  Please consider upgrading to new releases or helping to test
+release candidates.*
+
+- [Bitcoin Core 26.0rc2][] is a release candidate for the next major
+  version of the predominant full-node implementation. There's a [testing
+  guide][26.0 testing] available.
+
+- [Core Lightning 23.11rc3][] is a release candidate for the next
+  major version of this LN node implementation.
+
+<!-- FIXME:harding to update Tuesday -->
+
+## Notable code and documentation changes
+
+*Notable recent changes in [Bitcoin Core][bitcoin core repo], [Core
+Lightning][core lightning repo], [Eclair][eclair repo], [LDK][ldk repo],
+[LND][lnd repo], [libsecp256k1][libsecp256k1 repo], [Hardware Wallet
+Interface (HWI)][hwi repo], [Rust Bitcoin][rust bitcoin repo], [BTCPay
+Server][btcpay server repo], [BDK][bdk repo], [Bitcoin Improvement
+Proposals (BIPs)][bips repo], [Lightning BOLTs][bolts repo], and
+[Bitcoin Inquisition][bitcoin inquisition repo].*
+
+- [Rust Bitcoin #2213][] amends the weight prediction for P2WPKH inputs
+  processes during fee estimation. Since transactions with high-s signatures
+  are considered non-standard since [BIP66][], transaction building processes
```

Comment:
@murchandamus I'm going to make the change @vostrnad suggested (with some links added) since it maches my understanding.  Please feel free to revert if you disagree.

Reply:
Thanks, great